atc/step_recursor.go

```diff
@@ -51,6 +54,15 @@ func (recursor StepRecursor) VisitPut(step *PutStep) error {
 	return nil
 }
 
+// VisitRun calls the OnRun hook if configured.
+func (recursor StepRecursor) VisitRun(step *RunStep) error {
+	if recursor.OnPut != nil {
```

Comment:
```suggestion
	if recursor.OnRun != nil {
```

I know it's a WIP but just noticed this in passing in case it's hard to find later. 😛

Reply:
Fixed. Tnx for letting me know.